dashboard/test/ui/features/step_definitions/blockly.rb

```diff
@@ -60,11 +71,6 @@
   @browser.execute_script code
 end
 
-When /^I drag block matching selector "([^"]*)" to block matching selector "([^"]*)"$/ do |from, to|
```

Comment:
Is this step definition (and the one in the other file) no longer used anywhere?

Reply:
That's correct! Sorry, I should have mentioned that in the PR description.